stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.30.0
+version: 1.31.0
```

Comment:
If you make this 1.32.0 (something else just got merged ahead of this), then this will LGTM.